packages/dds/tree/src/core/tree/detachedFieldIndex.ts

```diff
@@ -106,6 +141,9 @@ export class DetachedFieldIndex {
 						0x7a9 /* Collision during index update */,
 					);
 					innerUpdated.set(minor, entry);
+					this.latestRelevantRevisionToFields
```

Comment:
Doesn't `this.latestRelevantRevisionToFields` need to be updated when `innerUpdated === undefined` is true?